katas/content/complex_arithmetic/complex_addition/Verification.qs

```diff
@@ -0,0 +1,54 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;    
+    open Microsoft.Quantum.Convert; 
+
+    operation ComplexAddExp(x : Complex, y : Complex) : Complex {
+    
+        // Calculate the sum of two complex numbers.  
+        let z = Complex (x::Real + y::Real, x::Imag + y::Imag);
+        return z;
+
+    }
+
+    operation ComplexRandom(min : Double, max : Double) : Complex{
```

Comment:
This operation and the next one are going to be used in multiple tests, so it makes sense to move them to a `common.qs` for this kata and include it in the dependencies - you can see examples in other katas

Reply:
Moved to common.qs